charts/placement-policy-scheduler-plugins/templates/rbac.yaml

```diff
@@ -91,14 +138,14 @@ roleRef:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: pp-plugins-scheduler-as-kube-scheduler
+  name: system:pp-plugins-scheduler-as-kube-scheduler
 subjects:
   - kind: ServiceAccount
     name: pp-plugins-scheduler
     namespace: {{ .Release.Namespace }}
 roleRef:
   kind: ClusterRole
-  name: pp-plugins-scheduler
+  name: system:pp-plugins-scheduler
```

Comment:
ditto (why `system` prefix)

Reply:
same, I followed the ClusterRole in the metrics-server